pycarol/bigquery.py

```diff
@@ -254,6 +255,31 @@ def _generate_client(service_account: T.Dict) -> bigquery.Client:
         client = bigquery.Client(project=project, credentials=credentials)
         return client
 
+    def _build_query_job_labels(self) -> T.Dict[str, str]:
+        label_values = [
+            self._env.get("env_id", ""),
+            self._env.get("env_name", ""),
+            self._env.get("org_id", ""),
+            self._env.get("org_name", ""),
+            "sync",
+            "py_carol",
+            "",
+            self._env.get("app_name", ""),
+            "",
+        ]
+        label_keys = [
+            "tenant_id",
+            "tenant_name",
+            "organization_id",
+            "organization_name",
+            "job_type",  ## Is any case that using this class is async?
+            "source",
+            "task_id",  ## In this case may we use the "mdmBigQueryProvisionId" from 'carol._current_env()'?
+            "carol_app_name",
+            "carol_app_process_name",  ## I do not understand what should be this property
```

Comment:
![image](https://github.com/totvslabs/pyCarol/assets/16036931/3bd4a4f4-c4e9-4503-883c-b869f2b37cab)
This property points to the Carol App process. Batch apps may have many processes while online apps usually have a single process. Operator sets environment variables before starting the processes. The variable you are looking for is ALGORITHM_NAME (see the image). You may use os package to get this value.